bounties/pip/computervision-recipes/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-12-09",
+    "AffectedVersionRange": "*",
+    "Summary": "Arbitrary Code Execution",
+    "Contributor": {
+      "Discloser": "B3EF",
+      "Fixer": ""
+    },
+    "Package": {
+      "Registry": "pip",
```

Comment:
I can't see that it belong to the PyPi registry, i.e. can be installed using pip.

Can you check if you can find the PyPi URL otherwise, please change the registry to `other` in the file format and for the `Registry` value in the `vulnerability.json`.

Reply:
https://github.com/microsoft/computervision-recipes/blob/master/SETUP.md
i have seen a pip based installation here , that's why i have added it to the pip registry
do i need to change it ?
<3